.github/workflows/linux.yml

```diff
@@ -71,6 +71,11 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.9'
+      - name: Install taplo
+        uses: baptiste0928/cargo-install@v2
+        with:
+          crate: taplo-cli
+          locked: true
```

Comment:
Will this be installed by `./mach bootstrap` below now?

Reply:
It would be, but this way it's faster as this action will cache it. Using cargo install takes 4-5min.